### DISCUSSION TITLE
Version Candidate 3.5.2 RC0

### DISCUSSION THREAD

**Discussion by ynaessens** [JeromeDevome/GRR#396] [General]
Evolution de GRR version 3.5.1, avec des correctifs et des ajouts :
-     ajout : afficher le formulaire de demande de réservation même si la connexion est obligatoire
-     ajout : possibilité d'afficher le formulaire de contact pour réservation dans la page de login
-     ajout : possibilité d'afficher un sélecteur de date direct dans le menu gauche
-     ajout : possibilité pour un gestionnaire de ressource de déclarer la ressource empruntée depuis la page de réservation
-     ajout : possibilité de définir une page d'accueil personnalisée
-     ajout : fonctionnalité Fichier joint, seuls les fichiers jpg, png, gif ou pdf sont acceptés en pièces jointes aux réservations
-     amélioration : fonctionnalité de limitation des réservations sur un intervalle de temps
-     amélioration : utilisation de requêtes SQL préparées
-     amélioration : affichage du site et du domaine dans le sélecteur de ressources
-     correctifs divers de code et vocabulaire
-     correctif : erreur d'ouverture de session dans le cas d'une authentification obligatoire
-     correctif : sécurité de l'URL
-     mise à jour jquery-UI en 1.14.0
-     mise à jour jquery en 3.7.1
-     modification : tri des réservations à modérer selon la date de début
-     modification : validation redirige vers view_entry

<hr /><em>This discussion was created from the release <a href='https://github.com/JeromeDevome/GRR/releases/tag/v3.5.2RC0'>Version Candidate 3.5.2 RC0</a>.</em>

**Comment by Philippe34** [comment]
Bonjour,
J'ai testé la version 3.5.2RC1. Merci pour ces améliorations et nouveautés, c'est bien sympa !
Tout va bien.
Il me reste toujours les mêmes bugs expliqués en #386 mais que je parviens à corriger partiellement dans week.php et day.php et qui rend l'outil fonctionnel dans mon cas.
Cordialement
